tfx/components/example_validator/component.py

```diff
@@ -91,10 +98,22 @@ def __init__(self,
           'been renamed to "statistics" and is deprecated. Please update your '
           'usage as support for this argument will be removed soon.')
       statistics = stats
-    anomalies = output or types.Channel(
-        type=standard_artifacts.ExampleAnomalies,
-        artifacts=[standard_artifacts.ExampleAnomalies()])
+    anomalies = output
+    if not output:
+      anomalies_artifact = standard_artifacts.ExampleAnomalies()
+      split_names = artifact_utils.get_single_instance(
+          list(statistics.get())).split_names
+      for split in split_names:
+        if exclude_splits and split in exclude_splits:
+          split_names.remove(split)
```

Comment:
there is a short way for this 

[i for i in alist if i not in blist]

https://www.geeksforgeeks.org/python-remove-all-values-from-a-list-present-in-other-list/

Reply:
yes, i saw this, but exclude_lists might be none so we need to check for that as well so it will be [i for i in alist if not blist or i not in blist]. 

with formatting, this will also take three lines, and it's looks more complicated.

should i still use the short way?